src/security/LockingPointer.h

```diff
@@ -28,13 +28,6 @@
 
 #endif /* USE_OPENSSL */
 
-// Macro to be used to define the C++ equivalent function of an extern "C"
-// function. The C++ function suffixed with the _cpp extension
-#define CtoCpp1(function, argument) \
```

Comment:
Moving this definition makes a src/store/SwapMetaOut.h [comment](https://github.com/measurement-factory/squid/blob/40d6518de7ee641983d5d92bb6d44f08a077c6e2/src/store/SwapMetaOut.h#L17C1-L17C1) about it stale. If this move is no longer necessary for this PR, please undo it. Otherwise, please update that stale comment.

Reply:
Reverted the change, CtoCpp1 is SSL-specific again. Resolving.

Reply:
> Reverted the change, CtoCpp1 is SSL-specific again. Resolving.

The change was not reverted. Unresolving.

Reply:
Moved to own file, thanks for calrifying what you meant